program/src/processor.rs

```diff
@@ -6652,7 +6652,7 @@ fn invoke_transfer<'a>(
 }
 
 #[inline(never)]
-fn read_oracle(
+pub fn read_oracle(
```

Comment:
unrelated/accident?

Reply:
We were using it in our onchain trading program which needed to read the oracle. It's unrelated to equity calc, but related to other programs importing mango lib